iron-data-table.html

```diff
@@ -665,6 +691,20 @@
         }
       },
 
+      _onChangeColumnOrder: function(e) {
+        this.columns.forEach(function(col) {
```

Comment:
Could you add some tests for this handler? Would also me to understand better how it works 😄